components/engine_rocks/src/snapshot.rs

```diff
@@ -30,6 +30,10 @@ impl RocksSnapshot {
             }
         }
     }
+
+    pub fn db(&self) -> Arc<DB> {
```

Comment:
Not necessary anymore, people can use `as_inner`.

Reply:
Done